boards/xtensa/esp32s_nodemcu/Kconfig.board

```diff
@@ -0,0 +1,8 @@
+# ESP32S NodeMCU board configuration
+
+# Copyright (c) 2022 Intel Corporation
```

Comment:
Changing the date to 2022 but it was not developed by intel? I suggest to rollback to original date, or put the author in the copyright.

Reply:
You can use as an example:  
```c
/*
 * Copyright (c) 2022 Your Name <your@email>
 *
 * SPDX-License-Identifier: Apache-2.0
 */
 ```